app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError
+        if visitor["vaccine"].get("expiration_date"):
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError
+        if (
+                not visitor.get("wearing_a_mask")
+                or visitor["wearing_a_mask"] is False
+        ):
+            raise NotWearingMaskError

```

Comment:
The `NotWearingMaskError` should include a descriptive message when raised. Consider using `raise NotWearingMaskError('All visitors must wear masks')`.